autocorpus/autocorpus.py

```diff
@@ -4,16 +4,27 @@
 from pathlib import Path
 from typing import Any
 
-from bioc import biocjson, biocxml
 from bs4 import BeautifulSoup
+from marker.converters.pdf import PdfConverter  # type: ignore
+from marker.models import create_model_dict  # type: ignore
+from marker.output import text_from_rendered  # type: ignore
+
+from autocorpus.bioc_supplementary import (
```

Comment:
This should also be a relative import (like the rest below)
```suggestion
from .bioc_supplementary import (
```